cluster/addons/kube-network-policies/kube-network-policies.yaml

```diff
@@ -45,6 +45,11 @@ spec:
         - mountPath: /lib/modules
           name: lib-modules
           readOnly: true
+        env:
+        - name: MY_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
```

Comment:
boo. you should make it a command-line option

Reply:
what is the benefit?

Reply:
independently, we can always add it , I just want to fix the flake

Reply:
I will move it to a flag https://github.com/kubernetes-sigs/kube-network-policies/issues/43